File: C/Atividade27.c
#include <stdio.h>
#include <conio.h>
#include <stdlib.h>
#include<stdio.h>
#include<time.h> 
#include<string.h>
#include<locale.h>  

int main()
{
	int num,mult,i,A,B;
    printf ("Insira um numero positivo:");
    scanf ("%d", &num);

	while (num<=0)
	{
		printf ("Invalido! Insira um numero positivo:");
    	scanf ("%d", &num);
	}	

    printf ("\n\nVoce digitou o numero positivo %d",num);
    
    
	printf ("\n\nDigite o intervalo da multiplicacao:\n");
	printf ("Digite o primeiro valor: ");
	scanf ("%d", &A);
	
	printf ("\nDigite um valor maior que %d: ",A);
	scanf ("%d", &B);
	
	while (A>=B)
	{
		printf ("\nInvalido! Digite um numero maior que %d!: ", A);
		scanf ("%d", &B);	
	}
	
	printf ("\n\nO intervalo da mutiplicacao decrescente de %d num intervalo de %d a %d e:\n\n",num, B, A);
	
	for (i=B;i>=A; i--)
	{
		mult=num*i;
		printf ("%d * %d = %d\n",num,i,mult);
	}	
    return 0;
}

File: Python/Atividade27.py
valor1 = float(input("Digite um valor positivo: "))

while (valor1<=0):
    valor1 = float(input("Invalido! Digite um número positivo: "))

print ("Digite o intervalo da tabuada:")

valor2 = int(input("Valor 1: "))

valor3 = int(input("Valor 2: "))

while (valor2>valor3):
    print ("Invalido! O Valor 2 deve ser maior que o valor 1! Digite novamente:")

    valor2 = int(input("Valor 1: "))
    valor3 = int(input("Valor 2: "))


print ("A tabuada de", valor1, "no intervalo de", valor3, "a", valor2, "é:" )

for i in range (valor3, valor2-1, -1):

    tabuada= valor1*i
    print (valor1,"*", i, "=", tabuada )